### ISSUE TITLE
Screen Reader Support

### ISSUE THREAD

**Issue by mrdrogdrog** [hedgedoc/hedgedoc#2903]
**Is your feature request related to a problem? Please describe.**
Blind or visually impaired people can't use hedgedoc 1.7 in a very useful way. There are keyboard traps, not labeled components and missing instructions.

**Describe the solution you'd like**
- Insert instructions, that are only visible for screenreaders (see bootstrap class `sr-only`)
- Add way more keyboard shortcuts for navigation (switch to editor + switch to output, toolbar shortcuts)
- Add ARIA-tags where possible.

**Additional context**
This should be done as one of the last issues, because the UI structure can still change a lot.

**Comment by ndarilek** [comment]
Wondering if there have been any updates on this?

I know CodeMirror was a blocker at one point, but V5 seems screen-reader-accessible. I'm a screen reader user attempting to use the hedgedoc-hosted demo and it doesn't seem to work well. I'm not familiar enough with this project to know whether that's running an older version or if it's fairly up-to-date.

Thanks.

**Comment by mrdrogdrog** [comment]
Hello @ndarilek ,

CodeMirror 5 is the old version that is used in HedgeDoc 1. But because we focus on the development of HedgeDoc 2, which will use CodeMirror 6, we won't back port CodeMirror 6 to HD1.

**Comment by mrdrogdrog** [comment]
Hedgedoc 2 is currently in development but you can test the new UI on https://hedgedoc.dev .

**Comment by ndarilek** [comment]
Got it, so V5 is the old inaccessible version running on HD 1.x, which is the version running on hedgedoc.org?

I'm looking for something like this as a blind screen reader user, so looks like 2.x should at least be usable where 1.x isn't. Is there a milestone or something to track progress on that?

Thanks.

**Comment by ndarilek** [comment]
Oh, this is *much* nicer. Thanks, looking forward to V2.

**Comment by mrdrogdrog** [comment]
Yes, there is a milestone "Version 2.0"
